src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -703,7 +703,7 @@ void program::cleanup() {
             }
         }
     }
-    _kernels_cache->reset();
+    // _kernels_cache->reset();
```

Comment:
Can we remove this only when serialize is used?

Reply:
Updated to preserve `_kernels_cache` only when exporting it. Thank you.